dds/DCPS/NetworkConfigUpdater.cpp

```diff
@@ -0,0 +1,160 @@
+/*
+ *
+ *
+ * Distributed under the OpenDDS License.
+ * See: http://www.opendds.org/license.html
+ */
+
+#include "DCPS/DdsDcps_pch.h" //Only the _pch include should start with DCPS/
+
+#include "NetworkConfigUpdater.h"
+
+#ifdef OPENDDS_NETWORK_CONFIG_UPDATER
+
+OPENDDS_BEGIN_VERSIONED_NAMESPACE_DECL
+
+namespace OpenDDS {
+namespace DCPS {
+
+NetworkConfigUpdater::NetworkConfigUpdater()
+{
+}
+
+bool NetworkConfigUpdater::open()
+{
+  if (DCPS_debug_level > 0) {
+    ACE_DEBUG((LM_DEBUG, "NetworkConfigUpdater::open enumerating interfaces"));
+  }
+
+  struct ifaddrs *ifap = 0;
```

Comment:
Declaring struct variables like this isn't necessary in C++ and the by the guidelines `*` and `&` should always be with the type.

```suggestion
  ifaddrs* ifap = 0;
```